WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,22 +1,18 @@
 RSpec.configure do |config|
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:truncation, except: %w(Countries Continents Events Rounds Formats teams))
   end
 
   config.before(:each) do
     DatabaseCleaner.strategy = :transaction
   end
 
   config.before(:each, js: true) do
-    DatabaseCleaner.strategy = :truncation
+    DatabaseCleaner.strategy = :truncation, {except: %w(Countries Continents Events Rounds Formats teams)}
```

Comment:
I don't think this is needed, since I don't have it and things work for me :)

I think this only cleans out things created since the beginning of the test, and preexisting data is unaffected.